src/SFA.DAS.QnA.Api.Views/Views/QuestionPartials/_CheckboxList.cshtml

```diff
@@ -55,7 +55,8 @@
 </div>
 
 <script type="text/javascript">
-    if (document.querySelector('.js-check-all-container')) {
+    // Is ".js-check-all-container" shown on the page
+    if (document.querySelector('.js-check-all-container').offsetParent) {
```

Comment:
This is definitely ok, is it? Lot of pages will be affected by this, but if you're sure... 💃

Reply:
Yes. This prevents the indent which is making all the checkboxes look incorrectly aligned in apply.